epersons.md

```diff
@@ -253,6 +253,7 @@ Status codes:
 * 201 Created - if the operation succeed
 * 400 Bad Request - if the email address didn't match the token or already exists. If the token doesn't exist or is expired
 * 401 Unauthorized - if the token doesn't allow you to create this account
+* 422 Unprocessable Entity - If password validation is enabled and the password of respects the rules configured in the regular expression
```

Comment:
I think the text of this could be improved.  Maybe say "If password validation is enabled, and the new password doesn't validate based on those rules.  See [configuration name]"  (Please replace the [configuration name] with the final name of the configuration... I didn't fill it out because I suggested it be renamed in my review here: https://github.com/DSpace/DSpace/pull/8404#pullrequestreview-1060958194)

I think we also need to add this 422 exception to the `Patch -> Add` section in this same `epersons.md` as it's also possible this will be thrown when changing your password.